tests/foreman/api/test_syncplan.py

```diff
@@ -607,8 +607,12 @@ def validate_repo_content(
             except AssertionError:
                 sleep(30)
         else:
-            raise AssertionError(
-                'Repository contains invalid number of content entities')
+            repo = repo.read_json()
```

Comment:
Hi @oshtaier, can you pls give more context on why _json should be avoided? I use it because I don't get the `last_sync` attribute from the output of read(). So I guess I'd some changes to nailgun to support this, though I'm not sure it is worth the effort.